src/server/game/Entities/Player/Player.h

```diff
@@ -2825,6 +2828,8 @@ class TC_GAME_API Player : public Unit, public GridObject<Player>
         uint32 manaBeforeDuel;
 
         WorldLocation _corpseLocation;
+
+        SceneMgr* m_sceneMgr;
```

Comment:
There is no reason to use a pointer here (and no need to new/delete it), just use `SceneMgr m_sceneMgr;`